src/List/ListIcon.test.js

```diff
@@ -0,0 +1,26 @@
+import List from './List';
+import { mount } from 'enzyme';
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+describe('<ListIcon />', () => {
+    const ListIcon = (
+        <List.Icon glyph='accept' />
+    );
+
+    test('create list Icon item', () => {
+        let component = renderer.create(ListIcon);
+        let tree = component.toJSON();
+        expect(tree).toMatchSnapshot();
+    });
+
+    describe('Prop spreading', () => {
+        test('should allow props to be spread to the ListIcon component', () => {
```

Comment:
tiny thing, this could say `List.Icon component` so it's not confused with the ListIcon thing right above